charts/ccd-docker-user-profile-importer/values.yaml

```diff
@@ -0,0 +1,18 @@
+importer:
+  userProfile:
+    image: 'hmctspublic.azurecr.io/ccd/user-profile-importer:latest'
+    memoryRequests: '512Mi'
+    cpuRequests: '250m'
+    memoryLimits: '2048Mi'
+    cpuLimits: '1500m'
+    users:
+    jurisdictions:
+    waitHosts: ""
+    waitHostsTimeout: 300
+    microservice: ccd_definition
+    verbose: "false"
+    userProfileDatabaseHost: "${SERVICE_NAME}-ccd-postgres"
```

Comment:
these should be revisited? All of this file

Reply:
can you please address this ? ${} notation might not work . Also do we need 
importer:
  userProfile: at root level. It made sense earlier when it was a big chart.

Reply:
fixed